io.github.purplehorrorrus.Meridius.yml

```diff
@@ -0,0 +1,38 @@
+app-id: io.github.purplehorrorrus.Meridius
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: meridius
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-download:create
+  - --filesystem=xdg-music:create
+modules:
+  - name: meridius
+    buildsystem: simple
+    build-commands:
+      - cp -a * /app/bin
+      - install -Dm644 $FLATPAK_ID.desktop /app/share/applications/$FLATPAK_ID.desktop 
+      - install -Dm644 $FLATPAK_ID.metainfo.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml
+      - install -Dm644 build/icons/256x256.png /app/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
+    sources:
+      - type: archive
+        url: https://github.com/PurpleHorrorRus/Meridius/releases/download/v2.4.11/meridius-2.4.11.tar.gz
+        sha256: 0f3028ed4fa2f04ad7f5f455da8ebc5567feaa92db8c9760dd7eb5637d21aa16
+      - type: script
+        dest-filename: meridius
+        commands:
+          - /app/bin/meridiusreborn --no-sandbox "$0"
+      - type: file
+        path: io.github.purplehorrorrus.Meridius.desktop
+      - type: file
+        path: io.github.purplehorrorrus.Meridius.metainfo.xml
+      - type: file
+        path: icons/256x256.png
```

Comment:
this the icon I'm talking about You install the one in the tarbal. So remove it from here.
```suggestion
```